recipe/meta.yaml

```diff
@@ -12,33 +12,38 @@ source:
         # Skips 3 tests (check_extension, check_sql_stmt, and check_get_normal_zoom_extension_load) that are failing on OS X.
         # Re-visit this once a new release is out.
         - skip_failing_test.patch  # [osx]
+        - makefile.vc.patch  # [win]
+        - config-msvc.patch  # [win]
+        - gg_extras.patch  # [win]
 
 build:
-    number: 7
-    skip: True  # [win]
+    number: 8
+    features:
+        - vc9  # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
 
 requirements:
     build:
-        - sqlite 3.9.*
+        - sqlite 3.13.*
```

Comment:
@gillins I guess that what you are missing is the python dependency for the `vc` features to work. (Not sure if that is the cause for the failed rerender though.)